contracts/test/proposals/Addresses.sol

```diff
@@ -0,0 +1,175 @@
+pragma solidity =0.8.13;
+
+contract Addresses {
+    mapping(string => address) public mainnet;
+
+    struct RecordedAddress {
+        string name;
+        address addr;
+    }
+    RecordedAddress[] private recordedAddresses;
+
+    constructor() {
+        // ---------- VOLT ADDRESSES ----------
```

Comment:
would it be possible to do a vm.label on all of these addresses? that way when an integration test fails, we would be able to immediately see a stack trace that shows the names of the different contracts involved during the test failure. I know there isn't a reference to a vm here, and wondering if this is feasible.

Reply:
we can do it, refactor the constructor to call `addMainnet(...)` instead of directly assigning to the mapping, and then add vm.label call in the function

good idea, it will be convenient for debugging